Uyuni proxy #10177 ke (#45)
uyuni-project/uyuni-docs: modules/installation/pages/install-proxy-uyuni.adoc

[[install-proxy-uyuni]]
= Install Uyuni Proxy with openSUSE

{productname} Proxy can be installed on openSUSE.

== Install Uyuni Proxy on openSUSE Leap 15.1

.Procedure: Installing openSUSE Leap 15.1 with Uyuni

> . As the base system, install openSUSE Leap 15.1 with all available package updates applied.

Reviewer: Word order.

Author: I'll commit a change separately.

> . Configure a resolvable fully quallified domain name (FQDN) with menu:yast[System > Netwok Settings > Hostname/DNS].

Reviewer: ```suggestion
. Configure a resolvable fully qualified domain name (FQDN) with menu:yast[System > Network Settings > Hostname/DNS].
```

> . Add the repository for installing the {productname} Proxy software as [systemitem]``root``:

Reviewer: Word order.

Author: more adjustments to be submitted separately.

> . Add the repository for installing the {productname} Proxy software as [systemitem]``root``:

Author: ```suggestion
. Add the repository for installing the {productname} Proxy software.
As [systemitem]``root``, enter:
```

+

// variable assignment to avoid overlong lines
// https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-4.0-POOL-x86_64-Media1/
+
----
repo=repositories/systemsmanagement:/
repo=${repo}Uyuni:/Stable/images/repo/Uyuni-Proxy-4.0-POOL-x86_64-Media1/
zypper ar https://download.opensuse.org/$repo uyuni-proxy-stable
----

> . Refresh metadata from the repositories as [systemitem]``root``:

Reviewer: Word order.

+

----
zypper ref
----

> . Install the pattern for the {productname} Server as [systemitem]``root``:

Reviewer: Word order.

+

----
zypper in patterns-uyuni_proxy
----

> . Reboot.

Reviewer: Reboot what?

> * For more information about the stable version of {productname}, see https://www.uyuni-project.org/pages/stable-version.html.

Reviewer: Remove bullets.

Author: But several one-sentence paras in a row are even worse, IMO.  It's a list, and that's why I think a formal list is fine. If we'd connect the two items and have one para, it would also look bad.  I do not know what would be the best.

Reviewer: fair enough :)

* For more information about the development version of {productname}, see https://www.uyuni-project.org/pages/devel-version.html.

> When this installation is complete, you can continue with {productname} setup.

Reviewer: ```suggestion
When the installation is complete, you can continue with {productname} setup.
```

[[install-proxy-uyuni]]
= Install {productname} Proxy with {opensuse}

{productname} Proxy can be installed on {opensuse}.

== Install {productname} Proxy on {opensuse} Leap 15.1

.Procedure: Installing {opensuse} Leap 15.1 with Uyuni

. As the base system, install {opensuse} Leap 15.1 with all available package updates applied.

. Configure a resolvable fully qualified domain name (FQDN) with menu:yast[System > Network Settings > Hostname/DNS].

. Add the repository for installing the {productname} Proxy software.
As [systemitem]``root``, enter:
+

// variable assignment to avoid overlong lines
// https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-4.0-POOL-x86_64-Media1/
+
----
repo=repositories/systemsmanagement:/
repo=${repo}Uyuni:/Stable/images/repo/Uyuni-Proxy-4.0-POOL-x86_64-Media1/
zypper ar https://download.opensuse.org/$repo uyuni-proxy-stable
----

> . Refresh metadata from the repositories as [systemitem]``root``:

Author: ```suggestion
. As [systemitem]``root``, refresh metadata from the repositories:
```

+

----
zypper ref
----

> . Install the pattern for the {productname} Server as [systemitem]``root``:

Author: ```suggestion
. As [systemitem]``root``, install the pattern for the {productname} Server:
```

> . As [systemitem]``root``, install the pattern for the {productname} Server:

Author: ```suggestion
. As [systemitem]``root``, install the pattern for the {productname} Proxy:
```

+

----
zypper in patterns-uyuni_proxy
----

> . Reboot.

Author: ```suggestion
. Reboot  the {productname} Proxy.
```

> USE IT AS TEMPLATE FOR PROXY INSTALL

Reviewer 2: Not sure why this is here.

Author: I forgot to remove it.

[[install-proxy-uyuni]]

> = Install Uyuni Proxy with openSUSE

Reviewer 2: FMPOV we should you use `{productname}`, {opensuse}, etc, everywhere. Same comment for the rest of the files.

Author: Unfortunately, it is not that clear whether macros in titles will work reliably, @jcayouette @Loquacity ? At least, in the past, we had issues...

Reviewer: They fail if it's the first word, from my testing.

Author: ok, I think I adjusted it everywhere where it will work.

{productname} Proxy can be installed on openSUSE.

> == Install Uyuni Proxy on openSUSE Leap 15.1

Reviewer 2: IIRC the version was now a variable, so we don't need to do a replace on all documents when we change to `15.2`

Author: [I think it would be even better, if we could avoid the version number here and at several other places completely.  And mention the version number somewhere in the intro.]

Unfortunately, I do not no where you must define those variables.

Reviewer 2: Maybe they are already defined. There are files at the root folder defining the stuff such as `{productname}`, etc. Check with the team if you have issues understanding how it works.

Author: ok, will check with the team.
